en/user_guide/websites_creation.rst

```diff
@@ -3,71 +3,89 @@ Website creation
-
 .. image:: ../../images/website_id.png
 
+The website id can be generated by Open Orchestra from the site name or filled by the user when
+creating the site.
+Once the site created, this field is no more editable.
+Open Orchestra verifies during the form validation that the site id does not already exist as it
+must be unique among all the sites.
+
 Aliases
 -------
+.. image:: ../../images/aliases.png
 
-Open Orchestra allows to create  several aliases for a site.
+A site may exist in different languages, and each of this versions could have minor differences, for
+instance a node could be published in English and not in French. Moreover, a site in English could be
+accessed from different domains. To make the managment of those problematics easier, Open Orchestra
+introduces the concept of site alias.
+Basically a site represents a set of site aliases sharing contents and pages, each site alias in a
+single language. Multiple site aliases can be configured with the same language, if they define
+different access points, eg domain + language prefix.
 
-For example :
+For instance, the site http://demo.open-orchestra.com/ exists in english and french. It presents two
+site aliases:
 
 * Scheme by default (required): http
-* Domain (required): demo.openorchestra.dev
+* Domain (required): demo.openorchestra.com
 * Language (required): English
 * Language prefix : en
 * main aliases : on
 
-The website home page will be ``http://demo.openorchestra.dev/en``.
-
-See the `routing documentation`_ for further information.
+* Scheme by default (required): http
+* Domain (required): demo.openorchestra.com
+* Language (required): French
+* Language prefix :
+* main aliases : off
 
-Open Orchestra uses only the main alias for the sitemap generation.
+The website home page for english version is ``http://demo.openorchestra.com/en`` and
+``http://demo.openorchestra.com`` for the french version.
 
-.. image:: ../../images/aliases.png
+See the `routing documentation`_ for further information.
 
 Blocks available
 ----------------
-
-This attribute lists all the blocks available for the nodes of this website.
-Only the selected blocks will be available during the node contribution.
-
 .. image:: ../../images/blocks_available.png
 
+This field lists all the blocks available to nodes contribution for this website.
+Only the selected blocks will be available.
+
 Default Theme (required)
 ------------------------
 
-The attribute theme is the default theme used by the website.
-Each node can also have its own theme.
+The field theme is the default theme used by the website.
+This setting can be overriden node by node in order to have a specific theme on a special node.
 
 Sitemap generation (required)
 -----------------------------
 
-The two folling attributes are used for the sitemap generation :
+The two following settings are related to the sitemap.xml generation. That file helps Search Engine
```

Comment:
I would not put the explanation on what the sitemap is created for at the same place that the rest of the sentence.

Would it be possible to put in a note just under ? or a link to what a sitemap is?